### DISCUSSION TITLE
formily vue生态共建群

### DISCUSSION THREAD

**Discussion by janryWang** [alibaba/formily#1520] [Ideas]
![772F8129-007C-448A-B4E2-23E6D459040A](https://user-images.githubusercontent.com/4060976/119926624-4b15d680-bfaa-11eb-8917-a6640b68b2ba.jpeg)

**Comment by muuyao** [comment]
二维码过期了呀，现在有啥进展么？

**Comment by MisicDemone** [comment]
![image](https://user-images.githubusercontent.com/10394516/122156338-59fdf380-ce9b-11eb-9b5f-4b16504b0dc7.png)

**Comment by pojuy673** [comment]
想入群了解下，方便再发下么？谢谢

**Comment by janryWang** [comment]
![840D9D7E-218B-4DB6-94F9-507E59B9F9CE](https://user-images.githubusercontent.com/4060976/124760458-5a058680-df63-11eb-8b64-42bb83d51e86.jpeg)

**Reply by fuzi1996** [reply]
二维码已过期,麻烦再来一个

**Comment by janryWang** [comment]
加我微信吧，注明：入Vue共建群
微信：janry_kk
